### ISSUE TITLE
"Enter password" screen - Back button does not quit the app

### ISSUE THREAD

**Issue by Qrysto** [NemProject/NEMAndroidApp#236]
Hi :)

When I'm idle for awhile then come back from the lock screen, the app asks for the password, "Enter password" screen shows up. Then if I press Back button (of the phone), the screen flashes once then come back to the "Enter password" screen. I think pressing Back button should quit the app in this case (or at least double pressing Back should).

Version: 1.0.51.b3315 BETA

Main net address: NBPL6F-J2OQGS-P6IJWZ-PMSL67-BQ6U3I-LIHVCA-DUO5

**Comment by Qrysto** [comment]
By the way, I think on main screen it should also take a double Back to quit the app

**Comment by fyahfox** [comment]
yes, back button from the password input screen should quit the app.

what is "main screen" for you? account list or dashboard of an account?

**Comment by jabo38** [comment]
Bounty paid. Tx hash: 4d4448544fd216b3eb1e0417912895fcc54c27801da8b9ad59683364599081a5

**Comment by Qrysto** [comment]
@fyahfox the account list. More generally wherever it takes one Back button to quit the app at the moment,

**Comment by MatCAlves** [comment]
At account list screen, a single tap in back button is enough to quit the app. But when at the password screen, that bug really is reproducible here. It doesn't matter how many taps on back button I do, the app never quits.

Version: 1.0.52.b3324

**Comment by fyahfox** [comment]
yes, i can reproduce it too. @am4n0w4r can you reproduce as well?

**Comment by kekkyojin** [comment]
Maybe are you running Android API 15 (Android 4.0.x), @Qrysto?
